### DISCUSSION TITLE
Regex literals

### DISCUSSION THREAD

**Discussion by EmilyGraceSeville7cf** [dotnet/csharplang#5129] [Language Ideas] [answered]
Hello! I want to know is it planned to add regex literals like [JavaScript ones](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions)?

**Comment by svick** [comment, marked as answer]
This is a duplicate of https://github.com/dotnet/csharplang/discussions/371. I'm not aware of any current plans for adding that feature.
